docs/topics/testing/tools.txt

```diff
@@ -34,12 +34,18 @@ short:
 * Use Django's test client to establish that the correct template is being
   rendered and that the template is passed the correct context data.
 
+* The :class:`~django.test.RequestFactory` is a trimmed down version of the test client that bypasses the routing and middleware layers
+  so you can test your view as you would test any other function in python with known inputs, 
+  testing for specific outputs. More detail on this can be found in :doc:`advanced testing topics</topics/testing/advanced>`.
```

Comment:
Wrap at 79 chars.

Reply:
I'm not sure if the last sentence adds value, as `` :class:`~django.test.RequestFactory` `` refers to exactly the same page :thinking: .